packages/trpc/server/routers/viewer/dsync/_router.tsx

```diff
@@ -0,0 +1,66 @@
+import authedOrgAdminProcedure from "../../../procedures/authedProcedure";
+import { router } from "../../../trpc";
+import { ZCreateInputSchema } from "./create.schema";
+import { ZDeleteInputSchema } from "./delete.schema";
+import { ZGetInputSchema } from "./get.schema";
+
+type DsyncRouterHandlerCache = {
+  create?: typeof import("./create.handler").createHandler;
+  get?: typeof import("./get.handler").getHandler;
+  delete?: typeof import("./delete.handler").deleteHandler;
+};
+
+const UNSTABLE_HANDLER_CACHE: DsyncRouterHandlerCache = {};
+
+export const dsyncRouter = router({
+  // Create directory sync connection
+  create: authedOrgAdminProcedure.input(ZCreateInputSchema).mutation(async ({ ctx, input }) => {
```

Comment:
These endpoints should only be accessed by org admins. Thanks @Udit-takkar @sean-brydon for reminding me about this